docs/General/createProfileACM.md

```diff
@@ -83,7 +83,9 @@ In this example, the hostname is **cb-vending1** and the DNS suffix is **burgerb
 
 10. Select the name of the **CIRA Configuration** you created previously from the drop-down menu.
 
-11. Click **Create.**
+11. Optionally, add **Tags** to help in organizing and querying devices as your list of managed devices grow. Type a tag, then present enter or comma to add as many as you like.
```

Comment:
"...add more tags separated by commas or press enter to complete."

Reply:
hmm, that seems to change the meaning to me since comma and enter perform the same action. Complete is when you click the create button.

Reply:
It was hard to determine what you were trying to communicate. I tried to interpret and then I missed the mark. Maybe take a second look at it.

Reply:
Actually, lets just remove this line since it doesn't make sense here. We'll just provide context for the tags, the UI tells them what to do.